.travis.yml

```diff
@@ -16,4 +16,4 @@ matrix:
     script:
       - make dev
     after_success:
-      - make travis_coverage
+      - travis_wait make travis_coverage
```

Comment:
I think we should keep it, I received a **build error** email yesterday because of **no outputs** within 10 minutes, and I am pretty sure it is **not** caused by bugs, sometimes travis machines become slow `make travis_coverage` may run more than 10 mintues.

BTW, `make travis_coverage` casts 15min+ in my machine.